content/understanding-react-rendering/index.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: "Understanding React Rendering"
+title: "How to Render React with optimization"
 date: "2020-09-23"
 coverImage: "index.png"
```

Comment:
Change the file name or here to reflect proper image file.

Reply:
Its done.